arm-monitor/2016-03-01/swagger/serviceDiagnosticsSettings_API.json

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
How is this different from `arm-monitor/2016-03-01/swagger/DiagnosticsSettings_API.json`?